newrelic-php-agent/templates/daemonset.yaml

```diff
@@ -20,17 +22,82 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
       containers:
-        - name: {{ .Chart.Name }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
-{{- with .Values.image.command }}
-          command:
-{{ tpl (toYaml .) $root | indent 12 }}
-{{- end }}
-          volumeMounts:
-            - mountPath: /tmp
-              name: newrelic
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.newrelic.image.repository }}:{{ .Values.newrelic.image.tag }}"
+        imagePullPolicy: {{ .Values.newrelic.image.pullPolicy }}
+{{- with .Values.newrelic.image.command }}
+        command:
+{{ tpl (toYaml .) $root | indent 10 }}
+{{- end }}
+{{- with .Values.newrelic.daemonset.extraEnv }}
+        env:
+{{ tpl (toYaml .) $root | indent 10 }}
+{{- end }}
+{{- with .Values.newrelic.daemonset.extraEnvFrom }}
+        envFrom:
+{{ tpl (toYaml .) $root | indent 10 }}
+{{- end }}
+{{- with .Values.newrelic.livenessProbe }}
+        livenessProbe:
+          {{- if $use_daemon_address }}
+          exec:
+            command:
+            - /bin/sh
+            - -c
+            - ls -al
+            - /tmp/.newrelic.sock
+          {{- else }}
+          tcpSocket:
+            port: {{ $.Values.newrelic.port }}
+          {{- end }}
+{{ tpl (toYaml .) $root | indent 10 }}
+{{- end }}
+{{- with .Values.newrelic.readinessProbe }}
+        readinessProbe:
+          {{- if $use_daemon_address }}
+          exec:
+            command:
+            - /bin/sh
+            - -c
+            - ls -al
+            - /tmp/.newrelic.sock
+          {{- else }}
+          tcpSocket:
+            port: {{ $.Values.newrelic.port }}
+          {{- end }}
+{{ tpl (toYaml .) $root | indent 10 }}
+{{- end }}
+{{- if $use_daemon_address }}
+        volumeMounts:
+          - mountPath: /tmp
```

Comment:
@cw-niinuma 
Does it work i `/var/run/newrelic.sock` is specified for `.Values.newrelic.address`?